### ISSUE TITLE
Deployment bugs & question

### ISSUE THREAD

**Issue by MikeKry** [OrchardCMS/OrchardCore#10233]
I have found few problems during bigger deployment:

1/ some older pages remained published and latest, even though there is newer version. I can not get rid of them even when re-publishing page.

2/ Deployment returns success even though it did not pass (e.g. returns error with duplicate autoroutepaths), there could be a failure description. Now I have to go into appdata to investigate what happened.

3/ deployment does not delete old content items, is it possible to setup deletion of content items which are no longer published on source environment?

Thanks

**Comment by hishamco** [comment]
I suggest to split this to smaller working items, so it will be easier for tracking. Could you please provide a sample deployment to reproduce the bugs that you are facing?

**Comment by sebastienros** [comment]
I would agree with creating more isolated issues, even though they all seem valid. I don't think it needs more repro for now, this seems quite explicit.

**Comment by MikeKry** [comment]
for context, we were deploying content after ~4 months from last deploy and it seems that errors 1 and 5 could be also affected by that, CMS was updated few days before new deploy.

I have also others related to tenants, but I will not post it in this issue, as it is big enough :) 

Should I split it, or will someone do it when he takes something from this task?

**Comment by sebastienros** [comment]
Yes, you, please split it in as many isolated issues as you can. You already did one, thanks. And close this "meta" one when you are done. Thanks.
